docs/getting-started/Usage.md

```diff
@@ -0,0 +1,47 @@
+# How to use warden #
+
+The warden service is the core process in warden. It's responsible for
+providing a way to grant vault tokens to [tokend][tokend]. warden is designed to run on every vault server that stores secrets.
+
+## Running warden ##
+
+The warden service is started by running the `bin/server.js` binary. The binary
+can be found in the folder where [warden is installed][installation]. The
+service blocks, running forever or until it's told to quit. The binary supports
+several [configuration options][Configuration].
+
+When starting warden you should see output similar to this:
+
+~~~text
+{"level":"info","message":"PKCS7 helper started. PID 10027. Port 9806","timestamp":"2016-09-06T14:31:20.421Z"}
+{"level":"info","message":"Listening on localhost:8080","timestamp":"2016-09-06T14:31:20.428Z"}
+~~~
+
+When running warden you should see output similar to this:
+
+~~~text
+authenticate request recieved
```

Comment:
Are some parts of the log, like this "authenticate request received" line, not formatted as JSON?